src/components/MainApp/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Form } from 'react-bootstrap';
+import { InputNewTodo } from '../InputNewTodo';
+import UserSelect from '../UserSelect';
+import { connect } from 'react-redux';
+import styles from './MainApp.module.css';
+
+
+type Todo = {
+    title: string,
+    user?: number,
+    isDone: boolean,
+}
+
+type MainAppProps = {
+    todos: Todo[],
+    addTodo: (t: Todo) => void,
```

Comment:
could use full name `todo: Todo` instead of `t: Todo` - more explicit and consistent with other param names, like: `todos: Todo[]`